mcsema/BC/Callback.cpp

```diff
@@ -660,6 +708,12 @@ llvm::Function *GetLiftedToNativeExitPoint(ExitPointKind kind) {
   loader.StoreReturnValue(
       block, ir.CreateCall(ir.CreateIntToPtr(pc, func_ptr_ty), call_args));
 
+  // This means that indirect call happened and caller pushed his return
```

Comment:
I think we want this *before* we call to the external, because the entrypoint/callback code does this:

```c++
   // Allocate any space needed on the stack for a return address.
   loader.AllocateReturnAddress(block);
```